doc/rtd/topics/cli.rst

```diff
@@ -199,6 +207,9 @@ aliases:
 .. code-block:: shell-session
 
     $ cloud-init query --list-keys
+
+.. code-block::
+
```

Comment:
Some of the examples in the query section (starting at line 222, and the section after that one) have a couple of issues:

1. Examples start with `%`, which I think aught to be standardized on `$` (not your doing).
2. They also strip the comment (`#`) in copying, which is not desirable.
3. Mixed command and output.

More mixed command and output in the status section of this page (again, since no changes there I can't put the comment there otherwise I would).